### PR TITLE
Add TEKZIPARK device

### PR DESCRIPTION
**Pull request by arif-mansoori**
#### Summary

Add TEKZIPARK device

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/tekzitel/tekzipark-profile-868.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 42
```
Comment (johanstokking) [inline vendor/tekzitel/tekzipark-profile-868.yaml:4]: This doesn't seem right.

Leave empty if you don't use one.

**Review by johanstokking — COMMENTED** [review]
Thanks!

Would it be possible to add codec examples?

File: vendor/tekzitel/tekzipark-profile-868.yaml (hunk 1 above)
Comment (arif-mansoori) [reply vendor/tekzitel/tekzipark-profile-868.yaml:4]: Yes, will add the examples and comment out the vendorProfileID

**Comment by arif-mansoori** [comment]
Added examples to the codec file and commented out vendorprofile ID

**Review by johanstokking — APPROVED** [review]
Perfect, thanks a lot

**Comment by johanstokking** [comment]
I rebased on `master` and resolved a conflict with another added vendor in the index. Other than that, no changes

**Comment by arif-mansoori** [comment]
Thanks Johan. I am unable to lookup the brand via Register end device on v3 cluster. Does this take time?

**Comment by johanstokking** [comment]
Yes, it will be part of a next release. We currently don't support automatic updating of the device repository yet.

**Pull request merged**